src/client/index.js

```diff
@@ -608,3 +613,33 @@ $('[data-bs-toggle="tooltip"]').tooltip();
 
 });
 
+
+let isAnyTabOpen = false;
+const codeTabId = 'editor-tab';
+const circuitTabId = 'circuit-tab';
+const codeTabBtn = $('#editor-tab-btn');
+const circuitTabBtn = $('#circuit-tab-btn');
+
+function openTab(tabId, element) {
+  $('.tab-wrapper').removeClass('open');
+  $(`#${tabId}`).addClass('open');
+
+  $('.tab-btn').removeClass('active');
+  $(`#${tabId}-btn`).addClass('active');
```

Comment:
There is some duplication here. Why not refrain from using ids, have some classes for both tabs and buttons, and make the change in one go?

Reply:
Id's are needed. Otherwise I am not able to guess which button was clicked and which tab to open. How am I supposed to check which of the two buttons was clicked if they share the same class?

Reply:
I think... I could simplify it this way, that the button and tab get the `active` class at the same time 🤔